### PR TITLE
Rlecomte/tdi 43510 my sql allow load local infile

### PR DESCRIPTION
**Pull request by rlecomte-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-43510
MySQL driver 8.0.18 needs a parameter names allowLocalLoadInfile so allow bulk output

**What is the new behavior?**
new parameter is added to jdbc urt

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tMysqlRow/tMysqlRow_begin.javajet [hunk 1]
```diff
@@ -23,9 +23,12 @@ imports="
 			}
 			if(dbproperties == null || ("\"\"").equals(dbproperties) || ("").equals(dbproperties)) {
 %>
-				String url_<%=cid%> = "<%=jdbcURL%>://" + <%=dbhost%> + ":" + <%=dbport%> + "/" + <%=dbname%>;
+				String url_<%=cid%> = "<%=jdbcURL%>://" + <%=dbhost%> + ":" + <%=dbport%> + "/" + <%=dbname%> + "?allowLoadLocalInfile=true";
 <%
 	        } else {
+				if (!dbproperties.contains("allowLoadLocalInfile")) {
+					dbproperties += "&allowLoadLocalInfile=true";
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMysqlRow/tMysqlRow_begin.javajet]: seems missing **\\"** here

Comment (rlecomte-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tMysqlRow/tMysqlRow_begin.javajet]: which line exactly ? I installed those 2 files in my Studio, th ejavajet compiled, the job code was generated successfully and the job completed flawlessly.

Comment (jzhao-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tMysqlRow/tMysqlRow_begin.javajet]: i also have another question:
Did you check whether this work with context ?

Comment (rlecomte-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tMysqlRow/tMysqlRow_begin.javajet]: First issue resolved. I'll check about the context issue.

Comment (rlecomte-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tMysqlRow/tMysqlRow_begin.javajet]: Context issue is not in the scope of this PR.

**Pull request merged**